conf/sender.cfg

```diff
@@ -24,8 +24,8 @@ use_ssl: true
 certificate: /etc/grid-security/hostcert.pem
 key: /etc/grid-security/hostkey.pem
 capath: /etc/grid-security/certificates
-# If this is supplied, outgoing messages will be encrypted
-# using this certificate
+# If supplied, outgoing messages will be encrypted using this certificate.
+# It must be the cert of the accounting repository that you are sending to.
```

Comment:
Maybe change `cert` to `certificate`?

Reply:
and maybe we could mention that this form of encryption is above and beyond encrypting the tunnel with SSL?

Reply:
Addressed, though I've also made it less accounting specific.